bin/add-feed.sh

```diff
@@ -0,0 +1,45 @@
+podcast_id=$1
+base_directory="../feeds"
```

Comment:
In bash programming naming conventions for variables is snake-bases uppercase. Refactor it.

Reply:
refactored